packages/frames-validator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xmtp/frames-validator",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
version updates should be done with a changeset

Reply:
Ahh I think this was my fault. It wasn't super clear to me from reading the readmes how to do releases.